common.js

```diff
@@ -34,6 +34,22 @@ const commonCss = `
     border: 1px solid rgb(230,230,230);
     border-radius: 2pt;
     text-align: center;
+    position: relative;
+  }
+
+  .samplingLoader {
```

Comment:
CSS class names should be kabob case, ie `sampling-loader`. Also, it's not really a "loader", but a "loading indicator". I would call it `loading-indicator`.

Reply:
I didn't write the code, just grabbed it from the bam1. But that's a good suggestion. I'll change it.